website/siteConfig.js

```diff
@@ -185,6 +185,14 @@ const siteConfig = {
 
     separateCss: ['static/static', 'static\\static'],
 
+    algolia: {
+        apiKey: '46ecd80046d78d4e5d9a5c06f559dfaa',
```

Comment:
Where did this key come from?  It feels a little unsafe to include a plaintext API key like this.

Reply:
@kevinbarabash It is provided Algolia. It is Search-only API Key, which is safe to use in front-end code and in production. Also, Docusaurus documentation suggests entering the API key to its configuration: https://docusaurus.io/docs/en/search.